.insomnia/Request/req_3429270d219b46f8b7c6590a0ec0c6ac.yml

```diff
@@ -0,0 +1,28 @@
+_id: req_3429270d219b46f8b7c6590a0ec0c6ac
+type: Request
+parentId: fld_80bc7ee1b14547dc860545b5af4878c5
+modified: 1690472414068
+created: 1690417154942
+url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/prisma"
+name: PRISMA
+description: ""
```

Comment:
Add description with a note about how it behaves differently depending on the feature flag.

Reply:
@OAbubakr I suggest that you create a separate PR because this one is blocked on other comments.